src/Switch/SwitchWrapper.jsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import { Switch } from './Switch';
+
+export const SwitchWrapper = ({ defaultToggleValue, small, disabled }) => {
+  const [isChecked, setIsChecked] = useState(defaultToggleValue || false);
+  return (
+    <Switch
+      currentValue={isChecked}
+      onChange={() => setIsChecked(!isChecked)}
+      id="test"
+      small={small}
```

Comment:
I think rather than having boolean's for this, we should have the concept of 'size' then we can have multiple values without extra props.

Reply:
Do you want me to write the CSS for the different sizes? or Just allow a size prop and if we need to expand it we can?

Reply:
Currently it is still a small boolean

Reply:
Cool, happy for it stay for now, but I think we will need to change in future to match style guide, but let's not get bogged down for now.